src/Json/JsonFile.php

```
@@ -2,37 +2,33 @@
-    private int|null $indent = null;
+
     /**
      * @psalm-var string[]
      */
     private static array $encodeArrayKeys = [];
+
     private static int $encodeIndent = JsonFormatter::DEFAULT_INDENT;
 
+    private int|null $indent = null;
+
+    public static function encode(mixed $data, int $options = 448, string $indent = self::INDENT_DEFAULT): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$indent'. (undefined)

(UnusedFormalParameter)